jekyll/_cci2/migrating-from-launch-agent-to-the-new-machine-runner-agent.adoc

```diff
@@ -0,0 +1,165 @@
+---
+contentTags: 
+  platform:
+  - Cloud
+---
+
+ = Migrating from Launch Agent to the new Machine Runner Agent
+:page-layout: classic-docs
+:page-liquid:
+:page-description: Instructions on how to migrate from Launch agent to the new machine runner agent
+:icons: font
+:toc: macro
+:toc-title:
+:machine:
+
+This page describes how to migrate an existing Launch Agent installation to the new Machine Runner Agent.
```

Comment:
```suggestion
This page describes how to migrate an existing launch agent installation to machine runner agent.
```

Reply:
Is it possible to add a sentence to say why people would want to do this, and how they can find out which agent they are currently using?